activesupport/test/cache/behaviors/cache_store_behavior.rb

```diff
@@ -566,6 +566,37 @@ def test_expires_in_and_expires_at
     assert_equal "Either :expires_in or :expires_at can be supplied, but not both", error.message
   end
 
+  def test_invalid_expiration_time_raises_an_error_when_raise_on_invalid_cache_expiration_time_is_true
+    ActiveSupport::Cache::Store.raise_on_invalid_cache_expiration_time = true
+
+    freeze_time
+    key = SecureRandom.uuid
+    error = assert_raises(ArgumentError) do
+      @cache.write(key, "bar", expires_at: 1.minute.ago)
```

Comment:
I think you could just pass `-60` directly to simplify the test. Gets rid of the freeze_time etc.